### PR TITLE
fix: fix powershell issues

### PR DESCRIPTION
**Pull request by sinedied**
## Purpose

- ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type

What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

- Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

- Test the code

```

```

## What to Check

Verify that the following are valid

- ...

### CODE REVIEW

File: scripts/index-data.ps1 [hunk 1]
```diff
@@ -19,4 +19,5 @@ npm ci
 npm run build --workspace=indexer
 
 Write-Host 'Running "index-files" CLI tool'
-npx index-files --wait --indexer-url "$env:INDEXER_API_URI" --index-name "$env:AZURE_SEARCH_INDEX" ./data/*.md
+$files = Get-Item "data/*.pdf"
```
Comment (shibbas) [inline scripts/index-data.ps1:22]: Should this be *.md?

**Comment by sinedied** [comment]
Yes definitely! Thanks for the catch

File: scripts/index-data.ps1 (hunk 1 above)
Comment (anfibiacreativa) [inline scripts/index-data.ps1:22]: Please @sinedied clarify, since this should be perhaps `data/*.md` for now...

**Review by anfibiacreativa — CHANGES_REQUESTED** [review]

File: scripts/index-data.ps1 (hunk 1 above)
Comment (sinedied) [reply scripts/index-data.ps1:22]: Seems like my previous reply didn't make it :/

Thanks for the catch, yes it should be *.md and I'll fix it

**Review by anfibiacreativa — APPROVED** [review]

**Pull request merged**